docs/strategies/timeout.md

```diff
@@ -82,9 +82,8 @@ HttpResponseMessage httpResponse = await pipeline.ExecuteAsync(
 
 ### Happy path sequence diagram
 
-![Timeout strategy - happy path](../media/timeout%20strategy%20happy%20path.png)
-
-<!-- ```mermaid
+```mermaid
+%%{init: {'theme':'dark'}}%%
```

Comment:
I don't use the dark theme, and these diagrams don't look good:

![image](https://github.com/App-vNext/Polly/assets/1439341/d4c9f7db-ab2a-43f8-bb93-b3f460123052)